lib/valkyrie/storage/fedora.rb

```diff
@@ -6,12 +6,10 @@ class Fedora
     PROTOCOL = 'fedora://'
 
     # @param [Ldp::Client] connection
-    def initialize(connection:, base_path: "/", fedora_version: 4)
+    def initialize(connection:, base_path: "/", fedora_version: 5)
```

Comment:
Use a constant for the default version because it is used elsewhere.

Reply:
Oh I see now. Yeah, that's reasonable. https://github.com/samvera/valkyrie/issues/720